docs/en/configuration.rst

```diff
@@ -150,6 +150,30 @@ setting ``seed_base_class`` in your config:
 
     seed_base_class: MyMagicalSeeder
 
+Custom Migration Template
+-------------------------
+
+Custom template for Migrations could be used either by defining template file path
+in configuration file:
+
+.. code-block:: yaml
+
+    templates:
+        file: src/templates/customMigrationTemplate.php.dist
```

Comment:
why do we need to call them .dist?
usually thats only needed if those end up in main repo as such and have to be copied manually to get activated.

Reply:
> why do we need to call them .dist? usually thats only needed if those end up in main repo as such and have to be copied manually to get activated.

Yes, You're right, just found this file in the code repository and decided to name it this way. I will fix it in a moment...